### PR TITLE
Group edit tiles

### PR DESCRIPTION
**Pull request by hugo-vrijswijk**

### CODE REVIEW

File: common/src/main/scala/explore/components/FormTimeSpanInput.scala [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import lucuma.core.util.Display
+import lucuma.core.util.TimeSpan
+import lucuma.core.util.TimeSpan.given
+import lucuma.react.common.ReactFnProps
+import lucuma.react.primereact.*
+
+import java.util.concurrent.TimeUnit
+import scala.collection.immutable.SortedMap
+
+case class FormTimeSpanInput(
```
Comment (hugo-vrijswijk) [inline common/src/main/scala/explore/components/FormTimeSpanInput.scala:22]: Should this be in a shared place like lucuma-ui?

Comment (rpiaggio) [reply common/src/main/scala/explore/components/FormTimeSpanInput.scala:22]: Yes, that sounds like a good idea.

File: common/src/main/scala/explore/components/FormTimeSpanInput.scala [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+          (acc + (unit -> result.toDouble), diff)
+      ._1
+
+  // Custom ordering to go from biggest to smallest
+  private given Ordering[TimeUnit] = Ordering.by[TimeUnit, Int](_.ordinal).reverse
+  private given Order[TimeUnit]    = Order.fromOrdering
+
+  private given Reusability[TimeSpan]                 = Reusability.byEq
+  private given [A: Eq]: Reusability[NonEmptyList[A]] = Reusability.byEq
```
Comment (rpiaggio) [inline common/src/main/scala/explore/components/FormTimeSpanInput.scala:86]: These are already in `lucuma.ui.reusability`.

File: common/src/main/scala/explore/components/FormTimeSpanInput.scala [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+  private given Display[TimeUnit] = Display.byShortName {
```
Comment (rpiaggio) [inline common/src/main/scala/explore/components/FormTimeSpanInput.scala:88]: Can we put this in a new package `lucuma.ui.display`?

File: explore/src/clue/scala/queries/common/GroupQueriesGQL.scala [hunk 4]
```diff
@@ -35,6 +35,14 @@ object GroupQueriesGQL:
         minimumRequired
         parentId
         parentIndex
+        ordered
+        minimumInterval {
```
Comment (rpiaggio) [inline explore/src/clue/scala/queries/common/GroupQueriesGQL.scala:39]: `$TimeSpanSubquery` ? (I didn't check but I'm assuming this is a `TimeSpan`).

File: explore/src/main/scala/explore/observationtree/AsterismGroupObsList.scala [hunk 5]
```diff
@@ -73,7 +73,7 @@ object AsterismGroupObsList:
   ): Callback =
     expandedIds.mod { expanded =>
       expanded
-        .exists(_ === obsIds)
+        .contains_(obsIds)
```
Comment (rpiaggio) [inline explore/src/main/scala/explore/observationtree/AsterismGroupObsList.scala:76]: Nice

**Review by rpiaggio — APPROVED** [review]
LGTM, with a suggestion for code rearrangement

**Pull request merged**